demos/ripples.html

```diff
@@ -12,7 +12,7 @@
       <svg width="{{width}}" height="{{height}}">
         <rect x="0" y="0" width="{{width}}" height="{{height}}" fill="#1B1B1B" />
 
-        {{#each points.items key="key" as |point|}}
```

Comment:
@GavinJoyce to clarify instead of changing key to @index make `points` supported by an Iterator that uses `length` and `forEach` in `RingBuffer` https://github.com/tildeio/glimmer/blob/master/packages/glimmer-test-helpers/lib/environment.ts#L137